src/app/presales/components/pre-sale-widget/index.tsx

```diff
@@ -118,6 +125,9 @@ export function PreSaleWidget({ token }: { token: PreSaleToken }) {
       return
     }
 
+    // FIXME: revert on launch
+    return
```

Comment:
Flagging for visibility

Reply:
Will have to be reverted tomorrow upon launching